installInstructions/buildCGNS.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/sh
+version= # TODO: PUT THE VERSION OF PETSC YOU WANT TO BUILD HERE (e.g 4.3.0)
+
+# If the user doesn't have an OPT_FLAGS environment variable, define the optimization flags here
+if [ -z ${var+x} ]; then
```

Comment:
Shouldn't this just be `-z $OPT_FLAGS`? Same for the other script

Reply:
Wow yeah the logic here was totally messed up, fixed now using [this stack overflow answer](https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash/17538964#17538964)